lib/utils/utils.ts

```diff
@@ -115,6 +115,18 @@ export const derivePairId = (pair: Pair) => {
   return `${pair.baseCurrency}/${pair.quoteCurrency}`;
 };
 
+/**
+ * Convert ticker symbole into pair.
+ * @param { string } pair - example: btc/ltc
+ */
+export const getPairId = (pair: string): Pair => {
```

Comment:
The name of this function is confusing, because it accepts what we refer to everywhere else as `pairId` but the name suggests that it returns `pairId`. I'm still of the opinion that we don't need this function since it's basically just an alias for the `split` function, but if we do want it in the future it should be called something like `getPairCurrencies`.